app/liquid_tags/slideshare_tag.rb

```diff
@@ -0,0 +1,33 @@
+class SlideshareTag < LiquidTagBase
+  def initialize(tag_name, key, tokens)
+    super
+    @key    = validate key
+    @height = 450
+  end
+
+  def render(_context)
+    finalize_html <<-HTML
+      <iframe
+        src="//www.slideshare.net/slideshow/embed_code/key/#{@key}"
+        alt="#{@key} on slideshare.net"
+        width="100%"
+        height="#{@height}"
+        frameborder="0"
+        scrolling="no"
+        allowfullscreen>
+      </iframe>
+    HTML
+  end
+
+  private
+
+  def validate(key)
+    if key.match?(/\A[a-zA-Z0-9]{14}\Z/)
+      key.strip
```

Comment:
```suggestion
      key
```

Key will be stripped from the first suggestion.